### DISCUSSION TITLE
The policy docs/wiki/ALZ-Policies.md

### DISCUSSION THREAD

**Discussion by mrunmayeeindap** [Azure/Enterprise-Scale#1377] [General]
Deploy Diagnostic Settings for Database for PostgreSQL to Log Analytics workspace is not working. It is throwing error attach error. Could you please check.

<img width="700" alt="image" src="https://github.com/Azure/Enterprise-Scale/assets/57096562/9a9b4b8a-8e98-4106-b431-61631ea97ba1">

**Comment by jtracey93** [comment]
will move this to the issue in #1378
